transcom/mymove
# B 17980 prime counselor approve reject email

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a869619)

This is for merging #11868 into main

## Summary

This branch sends an email to the customer once the Prime has completed counseling for a move.

### How to test

EDIT: Tae gave this much more concise walkthrough for getting a prime counseling move (the USMC part is what I was missing I think).

To get to the "Complete Counseling" step for prime in the simulator follow these steps:

1. Create a new move as Marine Corps -> origin duty location as "NS, Mayport, FL 32228"
2. Login as TOO in USMC Gblock
3. Under move task order tab -> edit orders to correct missing fields
4. Check necessary boxes for Counseling & shipment and approve/submit.
5. Go to prime simulator for the move and complete counseling.
